src/screens/Settings/index.tsx

```diff
@@ -451,14 +453,28 @@ function Settings() {
           )}
           <span className="save">{t('info.settings')}</span>
           <button
-            className="button is-text"
-            onClick={() =>
+            className={`button is-text ${
+              isCopiedToClipboard ? 'success-override-color' : null
+            }`}
+            onClick={() => {
               clipboard.writeText(
                 JSON.stringify({ appName, title, ...settingsToSave })
               )
-            }
+              // set copied to clipboard status to true if it's not already set to true
+              // used for changing text and color
+              if (!isCopiedToClipboard) {
+                setCopiedToClipboard(true)
+
+                setTimeout(() => {
```

Comment:
Maybe you can move this to a useEffect, it will clean this a bit (the onClick callback is getting kinda big) and you can fix a bug. The bug is that, if you click the copy to clipboard button and change to another screen before the timeout ends, you'll get an error in the console because react will try to update an unmounted component.

You can do something like:

```js
useEffect(() => {
  const timer = setTimeout(() => {
    setCopiedToClipboard(false)
  }, 3000)

  return () => clearTimeout(timer)
}, [copiedToClipboard])
```

the returned function will take care of stoping the timeout if needed